compass_app/app/lib/utils/view_command.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright 2022 Google LLC
```

Comment:
This is a file from the library that Stadia used for architecture. These are the "Command" types that the ViewModel can expose to the View. (I changed the based class of Command to ViewCommand, so it wouldn't clash. Otherwise, it's the copied file.)

I'm not sure if the commands currently in Compass app and these commands are compatible, since the ones in the Compass app are handling error logic and more. We don't need to call everything a command, I just wanted to show you this incase its helpful.

Reply:
I find this file really interesting because without even seeing the ViewModel, Component and View classes, I can start to see how it all works.